repl/repl.go

```diff
@@ -156,3 +199,74 @@ func printParserErrors(errors []string) {
 		fmt.Printf("%s", "\t"+msg+"\n")
 	}
 }
+
+// support for ABS init file
+const ABS_INIT_FILE = "~/.absrc"
```

Comment:
I think the logic for the ABS Init File should sit in a separate module than the repl. Maybe `runtime`?

Reply:
Don't want to overload `runtime` with golang package of same name. Let's try `init.go` instead.